Dockerfile

```diff
@@ -9,15 +9,42 @@ RUN apt-get update && apt-get upgrade -y
 
 RUN apt-get -y install --no-install-recommends \
   python-pip \
+	python3-pip \
   git
 
 RUN pip install --upgrade pip
 
 RUN pip install wheel
 RUN pip install --upgrade setuptools
+RUN pip3 install wheel
+RUN pip3 install --upgrade setuptools
 
 COPY . .
 
-RUN pip install -r requirements.txt
+RUN pip install -r requirements.txt -r test-requirements.txt
+RUN pip3 install -r requirements.txt -r test-requirements.txt
```

Comment:
Why installing both py and py3 here, is it necessary? Is backward compatibility support the only reason for doing this? 

I looked through the requirements.txt and test-requirements.txt files, only `mock==2.0.0` is a py2-only lib, but it became a stdlib in py3.

Reply:
I was doing it because (1) we ostensibly support both and so I wanted to test both (also travis tests both builds?)

Reply:
Then it makes sense!